kubelet/datadog_checks/kubelet/summary.py

```diff
@@ -0,0 +1,139 @@
+# (C) Datadog, Inc. 2018-present
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+from __future__ import division
+
+from datadog_checks.base.utils.tagging import tagger
+
+from .common import replace_container_rt_prefix, tags_for_docker, tags_for_pod
+
+
+class SummaryScraperMixin(object):
+    """
+    This class scrapes metrics from Kubelet "/stats/summary" endpoint
+    """
+
+    def __init__(self, *args, **kwargs):
+        super(SummaryScraperMixin, self).__init__(*args, **kwargs)
```

Comment:
I think you can remove that.

Reply:
Yep, kept it just in case, but in the end, nothing to do in there. Removed.